<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Amp_Mic_Quad_Mixer</class>
 <widget class="QDialog" name="Amp_Mic_Quad_Mixer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
     border: 1px solid #8f8f91;
     border-radius: 4px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }
QPushButton:checked{
	background-color: rgb(0, 178, 0);
}</string>
   </property>
   <widget class="QLabel" name="label_out1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N-Out1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut1_AIn3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_out2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N-Out2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut3_AIn2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut4_AIn3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut2_AIn2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut1_AIn2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut2_AIn4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut4_AIn2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut1_AIn1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut1_AIn4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut2_AIn1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_out3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N-Out3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut3_AIn4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_out4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N-Out4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_in1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A-In1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut4_AIn4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_in4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A-In4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut3_AIn3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_in3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A-In3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut4_AIn1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_in2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A-In2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut2_AIn3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="NOut3_AIn1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>35</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Close">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
